packages/spacecat-shared-data-access/test/unit/models/importer/import-job.test.js

```diff
@@ -25,6 +25,9 @@ const validImportJob = {
     enableJavascript: true,
     enableCss: true,
   },
+  initiatedBy: {
+    name: 'test',
```

Comment:
Thoughts on using `apiKeyName` instead of `name`? I feel like `initiatedBy.name` sounds like it is accessing a person's name.

Reply:
Done.